x-pack/test/security_solution_api_integration/test_suites/investigation/utils/timelines.ts

```diff
@@ -8,17 +8,37 @@
 import type SuperTest from 'supertest';
 import { v4 as uuidv4 } from 'uuid';
 import {
-  PersistNoteRouteRequestBody,
+  GetTimelinesResponse,
+  SavedTimelineWithSavedObjectId,
   TimelineTypeEnum,
 } from '@kbn/security-solution-plugin/common/api/timeline';
-import { NOTE_URL } from '@kbn/security-solution-plugin/common/constants';
-import type { Client } from '@elastic/elasticsearch';
-import { SECURITY_SOLUTION_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-server';
-import { noteSavedObjectType } from '@kbn/security-solution-plugin/server/lib/timeline/saved_object_mappings';
+import { TIMELINE_URL } from '@kbn/security-solution-plugin/common/constants';
+
+/**
+ * Deletes the first 100 timelines.
+ * This works in ess, serverless and on the MKI environments as it avoid having to look at hidden indexes.
+ */
+export const deleteTimelines = async (supertest: SuperTest.Agent): Promise<void> => {
+  const response = await supertest
+    // .get(`${TIMELINE_URL}?page_size=100&page_index=1&timeline_type=default`)
```

Comment:
Can this be removed?

Reply:
For sure, was doing some test and forgot to remove it, will do tomorrow!